### ISSUE TITLE
Emulate RA & DEC Traverse

### ISSUE THREAD

**Issue by RF52** [Stellarium/stellarium#1354]
When I'm "star hopping" on my telescope, I always like to traverse in just one direction at a time, RA or DEC. I can aim for a certain target, and monitor my arrival via eyepiece or webcam, independently of the setting circles. Providing that I know exactly what the target looks like.

Id like to be able to emulate this on Stellarium. Centre on a known object, traverse along the RA until adjacent to another object. Now, I know I can do this already - centre first target / turn on sensor view / switch to RA/DEC mount / press up or down arrow until I get there. BUT... Id like the appearance to be in ALT/AZI mode (in other words "what you see by eye") and I cannot find any way (setting telescope oculars to Equatorial mount or not) / (setting telescope mount to Equatorial or not). In other words, I'd like to see the image of both stars and sensor frame parallel to the horizon, but be able to actually move across the sky in RA and DEC directions.

I know that I can change the arrow keys from moving in RA/DEC to moving in ALT/AZI by switching the telescope mount (button at bottom of screen), but this also changes the screen view. What I want is to be able to change the function of the arrow keys, but retain the appearance of the screen.

I do hope this makes sense. I find it rather hard to articulate what I'm after. I've searched high and low for a way to do it, but cannot find anything. I'm hoping someone can put me right.

Excellent software by the way!

R.

**Comment by gzotti** [comment]
Stellarium is either in azimuthal mount mode with the screen top next to the zenith, or in equatorial mount mode with the screen top next to celestial north. There is no way currently to mix modes, and I will not be the one to change this, sorry.

**Comment by RF52** [comment]
I completely understand your reluctance! 

Is there an alternative approach - maybe via scripting? I have no experience of scripts with stellarium, but just a thought. 

R.


Sent from Samsung tablet


-------- Original message --------

**Comment by alex-w** [comment]
>Is there an alternative approach - maybe via scripting?

No, this is impossible implement via scripting. I think the better solution will be create a plugin for "star hopping" feature.

**Comment by alex-w** [comment]
See #460 for additional

**Comment by RF52** [comment]
Many thanks for the information. I’d love to offer to volunteer to help, but sadly I have neither the time nor the skills!

 

Sent from Windows 10.

**Comment by alex-w** [comment]
This is a good task for the community to participate in the contribution into Stellarium. Who wants to help us?

**Comment by RF52** [comment]
I'm currently struggling with health issues, and am sadly unable to offer to help at the present time.Thanks for keeping me in the loop.R.Sent from my Galaxy
-------- Original message --------From: "Alexander V. Wolf" ***@***.***> Date: 02/08/2022  17:55  (GMT+00:00) To: Stellarium/stellarium ***@***.***> Cc: RF52 ***@***.***>, Author ***@***.***> Subject: Re: [Stellarium/stellarium] Emulate RA & DEC Traverse (#1354) 
This is a good task for the community to participate in the contribution into Stellarium. Who wants to help us?
